### DISCUSSION TITLE
Is there a chiseled ASP.NET Core Runtime for Linux arm32?

### DISCUSSION THREAD

**Discussion by ali50m** [dotnet/dotnet-docker#4436] [Q&A]
I would like to build docker images for both linux/amd64 and linux/arm/v7(Raspbian). But I cannot find the chiseled 
 tag for arm32 on https://hub.docker.com/_/microsoft-dotnet-nightly-aspnet/.

**Comment by mthalman** [comment]
There is no chiseled image available for arm32. If you'd like to request this, please log an [issue](https://github.com/dotnet/dotnet-docker/issues).

**Comment by ali50m** [comment]
#4438
